### PR TITLE
Cowboy hats now have a (very slight) bullet resistance

### PR DESCRIPTION
**Pull request by BomberBro**
Now you can survive a shot during your high noon duel

:cl:
 * rscadd: Cowboy hats now protect very slightly from bullets

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Congratulations, you are now a git master :tm:

**Comment by Kharkov72** [comment]
this town aint big enough fer the two of us

**Comment by RaccoonBro** [comment]
yeehaw

**Comment by SonixApache** [comment]
🤠

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -425,7 +425,7 @@
 	desc = "Pefect for the closet botanist."
```
Comment (Kurfursten) [inline code/modules/clothing/head/misc.dm]: Not your fault but mind fixing this?

Comment (BomberBro) [reply code/modules/clothing/head/misc.dm]: Not at all

**Pull request merged**